components/feature/customtabs/src/main/java/mozilla/components/feature/customtabs/CustomTabWindowFeature.kt

```diff
@@ -33,7 +33,7 @@ class CustomTabWindowFeature(
     private val activity: Activity,
     private val store: BrowserStore,
     private val sessionId: String,
-    private val crashReporter: CrashReporter? = null
+    @VisibleForTesting internal val handleError: (Exception) -> Unit
```

Comment:
I don't know if this is what we want as a fix.  This failure to handle the link/intent is an A-C issue.

Reply:
I have found the issue and I have steps to reproduce in the Fenix issue - https://github.com/mozilla-mobile/fenix/issues/8412#issuecomment-592020541.

Seems like this is a Fenix issue and for fixing it we have to add new intent filter schemes.
Any touch on a link gets to AC which will try to open a client activity (in this case Fenix) and in our case Fenix doesn't know how to handle specific link schemes.

Because at the moment we can't easily add new intent filter schemes without a product decision the best we can do is:
1) Show a message to the user that his touch event on a specific link cannot be handled by the app (UX suggestion, you can check it here https://github.com/mozilla-mobile/fenix/pull/8903#issuecomment-595875216).
2) Log the error to our crashstats to better assess how many users are affected by this. (solution inspired by your patch).

Because the issue actually is in the AC client (it doesn't handle all schemes) AC will simply pass this error to the client and it will be its responsibility to handle this case. We on Fenix will use both of the above two solutions:
https://github.com/mozilla-mobile/fenix/blob/fdecc03b19b3f6b46874bafe14cb85fd278ee45a/app/src/main/java/org/mozilla/fenix/customtabs/ExternalAppBrowserFragment.kt#L87

Reply:
This is interesting.  I think this makes sense.